docs/cli_installation.md

```diff
@@ -22,6 +22,7 @@ brew install argocd
 
 ```bash
 curl -sSL -o argocd-linux-amd64 https://github.com/argoproj/argo-cd/releases/latest/download/argocd-linux-amd64
+mkdir -p /usr/local/bin/
```

Comment:
Should we also add a quick note that they should add this to their PATH if it isn't already there?

Reply:
agreed; (fixed)